crates/cubecl-runtime/tests/dummy/server.rs

```diff
@@ -16,32 +17,27 @@ use super::DummyKernel;
 /// The dummy server is used to test the cubecl-runtime infrastructure.
 /// It uses simple memory management with a bytes storage on CPU, without asynchronous tasks.
 #[derive(new, Debug)]
-pub struct DummyServer<MM = SimpleMemoryManagement<BytesStorage>> {
+pub struct DummyServer<MM = MemoryManagement<BytesStorage>> {
     memory_management: MM,
 }
```

Comment:
We could simply remove the generic parameter here.

Reply:
Nice missed that one